tools/tck-api/src/main/scala/org/opencypher/tools/tck/api/Scenario.scala

```diff
@@ -205,6 +216,10 @@ case class Scenario(categories: List[String], featureName: String, number: Optio
           throw ScenarioFailedException(msg, error)
       }
     }
+
+    def addParameters(additionalParameters: Map[String, CypherValue]): ScenarioExecutionContext = {
+      this.copy(parameters = this.parameters ++ additionalParameters)
```

Comment:
Shouldn't this do a check that the key sets of `parameters` and `additionalParameters` are disjoint?

Reply:
That sanity-check already lives in `ValidateSteps` now?